### PR TITLE
Derive: volume of heated body

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/volume_of_heated_body_from_initial_volume_and_heating_temperature.py [hunk 1]
```diff
@@ -27,7 +36,20 @@
...
+_volume = coef_def.volume
+_temperature = coef_def.temperature
+
+_eqn = coef_def.definition.subs(coef_def.volumetric_expansion_coefficient, expansion_coefficient)
+
+_volume_expr = dsolve(_eqn, _volume(_temperature), ics={_volume(0): start_volume}).rhs
+
+_volume_series = _volume_expr.series(_temperature, 0, 2).removeO()
+
+_volume_derived = _volume_series.subs(_temperature, final_temperature - start_temperature)
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/volume_of_heated_body_from_initial_volume_and_heating_temperature.py:50]: Why do we substitute with temperature difference? It should be final temperature IMO.

Comment (alesanter) [reply symplyphysics/laws/thermodynamics/volume_of_heated_body_from_initial_volume_and_heating_temperature.py:50]: Hm, I actually found a way to make this look sensible, will refactor it later.

**Comment by alesanter** [comment]
I don't understand why `mypy`'s failing. Everything looks good locally:
image

**Comment by blackyblack** [comment]
> I don't understand why `mypy`'s failing. Everything looks good locally:

Fixed

**Comment by alesanter** [comment]
> > I don't understand why `mypy`'s failing. Everything looks good locally:
> 
> Fixed

Thank you

**Review by blackyblack — APPROVED** [review]

**Pull request merged**